### DISCUSSION TITLE
Geomap: Text on map

### DISCUSSION THREAD

**Discussion by Fiorello8** [grafana/grafana#57020] [Dashboards, Panels & Visualizations]
Hi

i am using your awesome plugin!!

it would be very interesting display values ​​and texts in the map without using the tooltip function

My goal it would be display values ​​and texts at specific coordinates ...
perhaps it might be a good idea to see these text and numerical values ​​beyond a certain settable zoom level

Is it possible ?

**Comment by tedraymond** [comment]
For markers it is possible not for the zoom out part. I would like to see this for other layers as well

**Comment by Fiorello8** [comment]
I do not understand .. Can you explain better ? thanks in advance

Il giorno gio 27 ott 2022 alle ore 04:18 tedraymond <
***@***.***> ha scritto:

**Comment by tedraymond** [comment]
When marker layer is selected you can modify the Text Label
![image](https://user-images.githubusercontent.com/73015641/199836196-1c9da3c4-5d5f-4862-ad7b-e552d9178688.png)

Either a Fixed value or a result of your query.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
